scripts/package.lua

```diff
@@ -88,79 +98,85 @@
 	os.rmdir(pkgName)
 
 	print("Cloning source code")
-	z = os.executef("git clone .. %s", pkgName)
+	local z = execQuiet("git clone .. %s -b %s --recurse-submodules --depth 1 --shallow-submodules", pkgName, branch)
 	if not z then
 		error("clone failed", 0)
 	end
 
 	os.chdir(pkgName)
 
-	z = os.executef("git checkout %s", branch)
-	if not z then
-		error("unable to checkout branch " .. branch, 0)
-	end
+--
+-- Bootstrap Premake in the newly cloned repository
+--
 
-	z = os.executef("git submodule update --init")
+	print("Bootstraping Premake...")
+	if os.ishost("windows") then
+		z = execQuiet("Bootstrap.bat")
```

Comment:
Perhaps a prompt for the VS version could be done here?

Reply:
I will add an optional argument. I'd like this to be the base for an automated process so a prompt  would go against that.